examples/linux/assert/project.xml

```diff
@@ -1,4 +1,5 @@
 <library>
+  <!-- CI: run fail -->
```

Comment:
This is a hint to the `examples_compile.py` script that the example should be run and that a failure is expected.